modules/clear-with-shortcut/src/main/java/com/codeborne/selenide/clearwithshortcut/ClearWithShortcut.java

```diff
@@ -0,0 +1,93 @@
+package com.codeborne.selenide.clearwithshortcut;
+
+import com.codeborne.selenide.Driver;
+import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.commands.Clear;
+import com.codeborne.selenide.impl.WebElementSource;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import static org.openqa.selenium.Keys.DELETE;
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * Clean the input field value.
+ * <br>
+ * <p>
+ *  The standard Selenium method {@link WebElement#clear()} does not help in case of
+ *  "tricky" inputs generated by Vue.js, React and other fancy frameworks.
+ * </p>
+ * <br>
+ * <p>
+ *  That's why we need to clear the field value by emulating real user actions:
+ * </p>
+ * <ol>
+ *   <li>Select the whole text</li>
+ *   <li>press "Delete"</li>
+ * </ol>
+ */
+@ParametersAreNonnullByDefault
+public class ClearWithShortcut extends Clear {
+  private static final Logger log = LoggerFactory.getLogger(ClearWithShortcut.class);
+
+  @Nonnull
+  @CheckReturnValue
+  @Override
+  public SelenideElement execute(SelenideElement proxy, WebElementSource locator, @Nullable Object[] args) {
+    WebElement input = locator.findAndAssertElementIsEditable();
+    clearAndTrigger(locator.driver(), input);
+    return proxy;
+  }
+
+  /**
+   * Clear the input content without triggering "change" and "blur" events
+   */
+  @Override
+  public void clear(Driver driver, WebElement input) {
+    input.clear();
+    clearWithShortcut(driver, input);
+  }
+
+  protected void clearWithShortcut(Driver driver, WebElement input) {
+    Platform platform = getPlatform(driver);
+    if (!platform.isUnknown()) {
+      try {
+        CharSequence modifier = platform.modifierKey();
+        driver.actions()
+          .sendKeys(input, "0")
+          .keyDown(modifier).sendKeys("a").keyUp(modifier).sendKeys(DELETE)
+          .perform();
+      }
+      catch (WebDriverException failedToPressKeys) {
+        log.error("Failed to clear the input with shortcut", failedToPressKeys);
+      }
+    }
+  }
+
+  @Nonnull
+  @CheckReturnValue
+  protected Platform getPlatform(Driver driver) {
+    if (!(driver.getWebDriver() instanceof JavascriptExecutor)) {
+      return Platform.UNKNOWN;
+    }
+    try {
+      String platform = driver.executeJavaScript("return navigator.platform");
```

Comment:
Maybe it would be better if we use https://github.com/selenide/selenide/blob/f6e33870c540d4e91f4cbb29216676d9acba7d81/src/main/java/com/codeborne/selenide/impl/JavaScript.java#L29 here? It seems `JavaScript.java` handles rare cases like driver decorated with `WrapsDriver`

Reply:
Wow. 
1. Yes
2. Doesn't it mean we need to fix `driver.executeJavaScript` to also support `WrapsDriver` etc.?

Reply:
@BorisOsipov Fixed it in a separate PR: https://github.com/selenide/selenide/pull/1848